portfolio/src/main/webapp/script.js

```diff
@@ -106,32 +104,105 @@ function createCommentElement(comment) {
 
   return commentElement;
 }
-async function displayComments() {
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('maxNum').addEventListener('input', () => {
+    let parentElement = document.getElementById('comments');
```

Comment:
Is the goal here to clear the parentElement? This is one of the cases where innerHTML is ok,

you can set

parentElement.innerHTML = '';